webpack-intro/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "webpack-intro",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "jquery": "^3.6.0",
+    "webpack": "^5.72.1",
+    "webpack-cli": "^4.9.2"
```

Comment:
`webpack` and `webpack-cli` should be installed as dev dependencies.

To fix this, first uninstall `webpack` and `webpack-cli`
```
npm uninstall webpack webpack-cli
```

Reinstall them as dev dependencies
```
npm install --save-dev webpack webpack-cli
```